src/state-manager/TransactionQueue.ts

```diff
@@ -5146,19 +5146,19 @@ class TransactionQueue {
             }
 
 
-          if (this.usePOQo) {
+          // if (this.usePOQo) {
```

Comment:
is this a default setting now? can we confirm this with Andrew/Zach before removing this condition?

Reply:
Yes, this is the default setting. We have been removing non POQo branches of code.